lib/money/display_options.ex

```diff
@@ -47,27 +63,28 @@ defmodule Money.DisplayOptions do
       Application.get_env(:money, :strip_insignificant_fractional_unit, false)
 
     symbol = if Keyword.get(opts, :symbol, default_symbol), do: Currency.symbol(money), else: ""
-    symbol_on_right = Keyword.get(opts, :symbol_on_right, default_symbol_on_right)
-    symbol_space = Keyword.get(opts, :symbol_space, default_symbol_space)
-    fractional_unit = Keyword.get(opts, :fractional_unit, default_fractional_unit)
-    strip_insignificant_zeros = Keyword.get(opts, :strip_insignificant_zeros, default_strip_insignificant_zeros)
-    code = Keyword.get(opts, :code, default_code)
-    minus_sign_first = Keyword.get(opts, :minus_sign_first, default_minus_sign_first)
-
-    strip_insignificant_fractional_unit =
-      Keyword.get(opts, :strip_insignificant_fractional_unit, default_strip_insignificant_fractional_unit)
 
     %__MODULE__{
       separator: separator,
       delimiter: delimiter,
       symbol: symbol,
-      symbol_on_right: symbol_on_right,
-      symbol_space: symbol_space,
-      fractional_unit: fractional_unit,
-      strip_insignificant_zeros: strip_insignificant_zeros,
-      code: code,
-      minus_sign_first: minus_sign_first,
-      strip_insignificant_fractional_unit: strip_insignificant_fractional_unit
+      symbol_on_right: default_symbol_on_right,
+      symbol_space: default_symbol_space,
+      fractional_unit: default_fractional_unit,
+      strip_insignificant_zeros: default_strip_insignificant_zeros,
+      code: default_code,
+      minus_sign_first: default_minus_sign_first,
+      strip_insignificant_fractional_unit: default_strip_insignificant_fractional_unit
     }
   end
+
+  defp get_currency_custom_display_options(currency) do
```

Comment:
```suggestion
 @spec get_currency_custom_display_options(atom) :: Keyword.t()
  defp get_currency_custom_display_options(currency) do
```

Reply:
This is not a `Keyword.t()`. If I'm not wrong, this is a `map`.